apm-agent-plugins/apm-servlet-plugin/src/main/java/co/elastic/apm/agent/servlet/ServletApiAdvice.java

```diff
@@ -177,9 +181,22 @@ public static void onExitServletService(@Advice.Argument(0) ServletRequest servl
                 } else {
                     parameterMap = null;
                 }
+                Throwable exceptionAttribute = getThrowable(request);
```

Comment:
No need, since we can only use one
```suggestion
                if (t == null) {
                    t = getThrowable(request);
                }
```